cql/ManageCommonAbnormality.cql

```diff
@@ -582,25 +582,25 @@ define ReflexTestingRecommendationText:
   'Reflex testing with cervical cytology is indicated.'
 
 define Pap6MonthIntervalText:
-  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 6-month intervals.'
+  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 6-month intervals from date of most recent test.'
   
 define Pap1YearIntervalText:
-  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 1-year intervals.'
+  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 1-year intervals from date of most recent test.'
  
 define Pap3YearIntervalText:
-  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 3-year intervals.'
+  'If HPV testing or cotesting is not available, Pap testing may be performed. This is recommended at 3-year intervals from date of most recent test.'
   
 define TreatmentWithPregnancyConcernsRecommendationText:
   'If the patient is currently pregnant, endocervical curettage, endometrial biopsy, and treatment without biopsy are unacceptable.'
 
 define OneYearSurveillanceRecommendationText:
-  'Repeat testing in 1 year with primary HPV testing or cotesting is recommended.'
-
+  'Primary HPV testing or cotesting is recommended within 1 year of most recent test.'
+  
 define ThreeYearSurveillanceRecommendationText:
-  'Repeat testing in 3 years with primary HPV testing or cotesting is recommended.'
+  'Primary HPV testing or cotesting is recommended within 3 years of most recent test.'
   
 define FiveYearSurveillanceRecommendationText:
-  'Routine screening at 5-year intervals using primary HPV testing or cotesting is recommended.'
+  'Primary HPV testing or cotesting is recommended at 5-year intervals from date of most recent test.'
```

Comment:
Will adjust text to separate the next upcoming test and future intervals.

Reply:
Language now reads:
  Primary HPV testing or cotesting is recommended within 5 years of most recent test.
  Future testing is recommended at 5-year intervals.

This keeps the text consistent with the 1-year and 3-year surveillance recommendation text and makes clear the 5-year interval after.

Reply:
I'm not sure if there is a reason to separate `FiveYearSurveillanceRecommendationText` and `FiveYearSurveillanceIntervalText`. Will these ever be referenced independently from one another?

That would be my one remaining comment in this thread. Overall the updated language sounds a lot more clear, so this is not a critical change, since it doesn't affect what is visible to the user.

Reply:
@mrnosal and I discussed over a call, and decided it is fine to keep these recommendation text components separated, as they are in this PR.